components/ItemCard.js

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+export default function ItemCard({ name, date, quantity }) {
+  return (
+    <StyledItem>
+      <h3>{name}</h3>
+
+      <p className="qty">Qty:</p>
+      <div className="qty-value">{quantity}</div>
+
+      <p className="date">date:</p>
+      <div className="date-value">{date}</div>
```

Comment:
```suggestion
      <p>Qty:</p>
      <div>{quantity}</div>

      <p>date:</p>
      <div>{date}</div>
```
Please try and avoid using classNames to address different styled components in another component. 
Mixing the two things (styled components and classes) is usually considered bad practice.
The best is to just create new Styled Components e.g. `const Quantity = styled.p`